test/system/tools/fds/tests.py

```diff
@@ -57,10 +57,19 @@ def test_fast_discovery_closure(fast_discovery_tool):
     # sleep to let the server run
     time.sleep(1)
 
-    # On launching error report
+    # An exit code of 0 means everything was alright
```

Comment:
It would be great to have an expanded version of this comment explaining the possible return codes and their meaning